docs/sla.md

```diff
@@ -0,0 +1,93 @@
+
+### Measurement Methodology
+
+Terms used herein but not otherwise defined shall have the meanings ascribed to them in the Master Services Agreement. For purposes of this SLA, the following terms have the meanings set forth below:
+
+* “Availability” is calculated by subtracting from 100% Downtime Percentage.
+
+* “Downtime” refers to any period of five (5) minutes or more in which none of the Customer's end users can access Customer's assets or properties.
+
+* “Downtime Percentage” refers to the amount of downtime as a percentage of the total time in that month.
+
+### Service Credits
+
+If the Company fails to meet the above availability level, Customer will be entitled to a service credit equal to 0.7% of the Customer domain’s monthly recurring charge (MRC) for every 0.05% (22.5 minutes/month) of service unavailability (“Service Unavailable Event”).
+
+### Maximum Credits
+
+The total maximum number of service credits to be issued to Customer for any and all Downtime Periods that occur in a single billing month will not exceed 50% of the amount due by Customer for the Service(s) for the applicable month.
+
+### Claim of Credits.
+
+To submit a claim for Credits, Customers shall open a support ticket in the Company technical support ticketing system no later than ten (10) calendar days (240 hours) after the time when the Service Unavailable Event occurred and provide detailed descriptions of the Service Unavailable Event. The descriptions must include the duration of the Service Unavailable Event; supporting evidence such as logs, network traceroutes, and monitoring logs of the site(s) affected; and detailed descriptions about any attempts made by Customer to resolve the Service Unavailable Event. The support ticket must include the words “service credit request” in the ticket or email subject line. Granting of service credits is contingent upon the Customer having opened a support ticket with the Company. The duration of the downtime will be based upon both Parties’ internal records. Company has the ultimate right of decision in case of any dispute.
+
+Credits will be applied to the Customer’s bill for the Service during the billing cycle following the month in which the credit was granted. The award of service credits is the Customer’s sole remedy and the Supplier’s sole liability for any Service Unavailable Event or loss of service, and shall not exceed the monthly recurring charge paid by Customer for the Services in any event.
+
+This total credit is a sole remedy and not to exceed 100% of the monthly recurring charge of the respective month. The service credits granted on any day shall not exceed the service fee of that affected day in any event.				
+### Excluded events
+
+Notwithstanding the foregoing, no service level commitment or service level credit will be made if failure to meet one or more of the above service levels is attributable to any of the following events or circumstances:
```

Comment:
You refer to "one or more", but as far as I can see, there is only one defined earlier.

Reply:
No comment.